### ISSUE TITLE
This project is defunct: please prove me wrong :)

### ISSUE THREAD

**Issue by boxabirds** [MaurycyLiebner/enve#336]
This project looks amazing! But sadly it looks like it's defunct as there's been no activity for more than a year or so. Fingers crossed I'm wrong about that though! Maybe people know of viable alternatives?

**Comment by doc-halo** [comment]
Yes, it's dead. There are other open source options out now: friction2d, synfig, glaxnimate, tupi. They are actively developed although still at a slower pace than wanted, since more developers are needed. Friction2d is probably closer to enve in look and feel.

**Comment by hoijui** [comment]
- https://friction.graphics/
- https://www.synfig.org/
- https://glaxnimate.mattbas.org/
- https://github.com/xtingray/tupitube.desk (there is a software from KDE with the same name and purpose, but that one was abandoned 10 years ago)
- https://www.blender.org/features/story-artist/ (Blender for 2D animation)
- https://docs.krita.org/en/user_manual/animation.html (Krita)
- https://www.pencil2d.org/
- https://tahoma2d.org/

and of course:
https://alternativeto.net/software/enve/?license=opensource

**Comment by hoijui** [comment]
Friction2d is a direct fork of Enve, see here:
https://github.com/friction2d/friction
